hedyorg/hedy
# [FIX] Change font size dynamically based on username length

**Description**
In this PR we fix an issue where a too long username would mess up the layout of the explore page. By changing the font-size dynamically based on the username length the problem is solved.

**Fix for**
This PR fixes #1712.

**How to test**
Navigate to the explore page and make sure that a program with a long username is shared. Notice that the font of the longer username is smaller and, when still too long, truncated.
